### ISSUE TITLE
Flipped cents amounts from floatToAmount

### ISSUE THREAD

**Issue by mwcbrent** [ikr/money-math#37]
When passing single digit suffix amounts (pennies) they are flipped when passed to `floatToAmount` ex: 

- `floatToAmount(125.01)` yields `125.01`
- `floatToAmount(98.08)` yields `98.80`

I added these tests to the specs and they seem to return the consistent failure:

```
        it("doesn't flip 125.01 to 125.10", function () {
            assert.strictEqual(money.floatToAmount(125.01), "125.01");
        })

        it("doesn't flip 99.08 to 99.80", function () {
            assert.strictEqual(money.floatToAmount(99.08), "99.08");
        })
```

It seems like lines 135 - 137:

```
suffix = suffix.substring(0, 3);  // 010
var a = parseInt(suffix.substring(0, 3), 10); //10
var b = parseInt(suffix.substring(2, 3), 10);  //0
a = parseInt(String(a).substring(0, 2), 10); // 10
```
Which flips the suffix that is returned.

**Comment by ikr** [comment]
😨 Must be a regression introduced in v2.6.1. Will check it today. Thanks a lot for reporting! <3

**Comment by ikr** [comment]
Fixed in v2.6.2
